x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_settings.tsx

```diff
@@ -96,6 +99,23 @@ export const RestoreSnapshotStepSettings: React.FunctionComponent<StepProps> = (
           </EuiButtonEmpty>
         </EuiFlexItem>
       </EuiFlexGroup>
+      {dataStreams?.length ? (
+        <>
+          <EuiSpacer size="m" />
+          <EuiCallOut
+            iconType="help"
+            title={i18n.translate(
+              'xpack.snapshotRestore.restoreForm.stepSettings.dataStreamsCallout.title',
+              { defaultMessage: 'Backing indices' }
+            )}
+          >
+            <FormattedMessage
+              id="xpack.snapshotRestore.restoreForm.stepSettings.dataStreamsCallout.body"
```

Comment:
Based on the [guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types), "body" isn't an acceptable identifier. How about "description"?